### DISCUSSION TITLE
Filter by keyword or by "fiel" <--missing "d"

### DISCUSSION THREAD

**Discussion by jdshkolnik** [community/community#4938] [Projects and Issues] [answered]
![image](https://user-images.githubusercontent.com/2047520/128426431-d6a82493-5262-4ec7-9b53-3f8ead12e4f8.png)

**Comment by azenMatt** [comment, marked as answer]
Which browser is this in? 

looks like we caught this one, but please let me know if you are still seeing it

**Reply by jdshkolnik** [reply]
Chrome. Fixed.
